.github/workflows/CI.yml

```diff
@@ -504,39 +530,31 @@ jobs:
         uses: actions/cache@v2
         with:
           key: "cached_packages_${{ github.run_id }}"
-          path: build/packages
+          path: $PACKAGING_ROOT
 
       - name: Create SSM package
         run: |
-          ssm_pkg_version=`cat build/packages/VERSION`
+          ssm_pkg_version=`cat $PACKAGING_ROOT/VERSION`
           (aws ssm describe-document --name "testAWSDistroOTel-Collector" --version-name ${ssm_pkg_version} >/dev/null 2>&1) || {
             pip3 install boto3
             python3 tools/ssm/ssm_manifest.py ${ssm_pkg_version}
-            aws s3 cp build/packages/ssm s3://aws-otel-collector-test/ssm/${ssm_pkg_version} --recursive
+            aws s3 cp $PACKAGING_ROOT/ssm s3://aws-otel-collector-test/ssm/${ssm_pkg_version} --recursive
             python3 tools/ssm/ssm_create.py --no-default testAWSDistroOTel-Collector ${ssm_pkg_version} aws-otel-collector-test/ssm/${ssm_pkg_version} us-west-2
           }
 
       - name: Upload to S3 in the testing stack
         if: steps.e2etest-release.outputs.cache-hit != 'true'
         run: s3_bucket_name=aws-otel-collector-test upload_to_latest=0 bash tools/release/image-binary-release/s3-release.sh
 
-      - name: Login to Public ECR
-        if: steps.e2etest-release.outputs.cache-hit != 'true'
-        id: login-ecr
-        uses: docker/login-action@v1
-        with:
-          registry: public.ecr.aws
-          username: ${{ secrets.INTEG_TEST_AWS_KEY_ID }}
-          password: ${{ secrets.INTEG_TEST_AWS_KEY_SECRET }}
-        env:
-          AWS_REGION: us-east-1
-
-      - name: upload to ECR
+      - name: Find if adot collector image exist in public ecr
```

Comment:
Would it make sense to remove the `packaging-image` job and just have it done here instead? `e2etest-preparation` doesn't actually have a dependency on that job and `packaging-image` is a misleading job anyway.

Reply:
Make sense. Mostly I thought that ```packaging-image``` would be sufficient for building the image and push it to the publish ECR. Then for the ```e2etest-preparation```, we only need to check whether the image is exist or now.